### ISSUE TITLE
Possible precedence problem between ! and string eq

### ISSUE THREAD

**Issue by eserte** [pmqs/BerkeleyDB#13]
With perl 5.41.x there are new warnings:
```
Possible precedence problem between ! and string eq at /home/e/eserte/.cpan/build/2024101308/BerkeleyDB-0.65-1/blib/lib/BerkeleyDB.pm line 829.
Possible precedence problem between ! and string eq at /home/e/eserte/.cpan/build/2024101308/BerkeleyDB-0.65-1/blib/lib/BerkeleyDB.pm line 997.
```

**Comment by pmqs** [comment]
Thanks. Those are trivial issues to fix

**Comment by pmqs** [comment]
Fixed in 0.66.
